packages/grafana-prometheus/src/querybuilder/components/LabelFilterItemCombobox.tsx

```diff
@@ -0,0 +1,170 @@
+// Core Grafana history https://github.com/grafana/grafana/blob/v11.0.0-preview/public/app/plugins/datasource/prometheus/querybuilder/components/LabelFilterItem.tsx
+import { useCallback, useRef } from 'react';
+
+import { selectors } from '@grafana/e2e-selectors';
+import { AccessoryButton, InputGroup } from '@grafana/experimental';
+import { Combobox, ComboboxOption } from '@grafana/ui';
+
+import { QueryBuilderLabelFilter } from '../shared/types';
+
+function toComboboxOption(value: string): ComboboxOption {
+  return { value };
+}
+
+export interface LabelFilterItemProps {
+  defaultOp: string;
+  item: Partial<QueryBuilderLabelFilter>;
+  onChange: (value: Partial<QueryBuilderLabelFilter>) => void;
```

Comment:
I've changed this to `Partial<QueryBuilderLabelFilter>` because that is what it actually is!